### PR TITLE
sensory stim after tuesday review

### PR DESCRIPTION
**Pull request by mdovgialo**

### CODE REVIEW

**Comment by mdovgialo** [comment]
@achabuda 
@mroja 
@develancer 
Something like this?

File: obci/devices/haptics/HapticsControl.py [hunk 1]
```diff
@@ -33,7 +33,23 @@
...
+try:
+    from pyftdi.ftdi import Ftdi
+except ImportError as e:
+    msg = r'''
+    
+    
+    You have missing libraries:
+    pyftdi 0.11.3 and pyusb 1.0.0.b2
+    to install run:
+    sudo pip install --pre pyusb==1.0.0.b2
```
Comment (mroja) [inline obci/devices/haptics/HapticsControl.py]: in Debian & Ubuntu python-usb package is a better choice than pip

**Comment by mroja** [comment]
LGTM, after replacing pip install pyusb with apt-get install python-usb

**Comment by mdovgialo** [comment]
I'm not sure if that will be compatable. In pip I use next beta or release candidate for pyusb, in ubuntu it will be older and I think that wont work.

**Comment by mroja** [comment]
I should be ok in Ubuntu. In PISAK on 15.10 we use packaged python-usb and it works great.

**Comment by mdovgialo** [comment]
Changed.

**Pull request merged**